### ISSUE TITLE
Marking an event if one ep has a VOC

### ISSUE THREAD

**Issue by Marko-ilmkreis** [hzi-braunschweig/SORMAS-Project#4978]
### Situation Description
An event becomes more significant when a VOC is detected at an ep. Up to now, we have prefixed all events that this concerns with the prefix "VOC". 

### Feature Description

If one ep has an VOC, the event should be marked as VOC-event. 

### Possible Alternatives

Leave it like it is.

**Comment by bernardsilenou** [comment]
Since this is disease and can also be country specific,  suggest we add this as a filter on the event directory and not directly as an attribute to the event.
We can call it `extended filter` show   all filters not directly related to an attribute of the event.

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou if this is possible, that would be a good solution. For my understaning, there would be a filter that shows all events, where an ep has a VOC? Does this extract the data from the cases as ep as well?

**Comment by bernardsilenou** [comment]
@Marko-ilmkreis  Yes please. In this way we can add more filters related to the cases add contacts that are linked to the events.

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou to add that filter would be awesome.

**Comment by kwa20** [comment]
@bernardsilenou @Marko-ilmkreis I'd also like this approach since it follows what is planned for the contact directory in association to source cases and provides consistency. It might actually be hard to define the VOC's since they're DE specific so I'd suggest to add the disease variant filter as a subsequent filter of the disease.

**Comment by bernardsilenou** [comment]
@kwa20  That is right, we can add to case and event directories, all as extended filters, right?
